### PR TITLE
Automate settings

### PR DESCRIPTION
**Pull request by sbharadwajj**
automated settings for `secondorder_settings.py` and `firstorder_settings.py`

### CODE REVIEW

File: test/extensions/automated_settings.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 ###
 
 
-def make_simple_cnn_setting(act_cls, bias):
+def make_simple_act_setting(act_cls, bias):
```
Comment (f-dangel) [inline test/extensions/automated_settings.py:9]: Is this function still used, or can it be removed?

File: test/extensions/secondorder/diag_ggn/diaggnn_settings.py [hunk 2]
```diff
@@ -7,20 +7,23 @@
...
 SHARED_SETTINGS = SECONDORDER_SETTINGS
+
+
 LOCAL_SETTINGS = []
 
+
 ###############################################################################
 #                         test setting: Activation Layers                     #
 ###############################################################################
 activations = [ELU, SELU]
```
Comment (f-dangel) [inline test/extensions/secondorder/diag_ggn/diaggnn_settings.py:23]: no `LogSigmoid` here?

File: test/extensions/automated_settings.py (hunk 1 above)
Comment (sbharadwajj) [reply test/extensions/automated_settings.py:9]: It is in use for the activation settings.

File: test/extensions/secondorder/diag_ggn/diaggnn_settings.py (hunk 2 above)
Comment (sbharadwajj) [reply test/extensions/secondorder/diag_ggn/diaggnn_settings.py:23]: No, we have it in `secondorder_settings.py`

Comment (sbharadwajj) [reply test/extensions/secondorder/diag_ggn/diaggnn_settings.py:23]: https://github.com/f-dangel/backpack/blob/f362a4c67551f033d5a8be208d8566421bca36ac/test/extensions/secondorder/secondorder_settings.py#L95 ; here

**Pull request merged**